### PR TITLE
Frontend localization actors

### PR DESCRIPTION
**Pull request by wintondeshong**
While there will be more utilities to come, here are some initial actors necessary.

Fixes issue #10

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,61 +1,65 @@
+import { CultureParams } from "./interfaces/culture-params";
```
Comment (brandongregoryscott) [inline src/index.ts:1]: Is this meant to be an import? 😛

File: src/utilities/route-utils.test.ts [hunk 2]
```diff
@@ -210,6 +210,78 @@ describe("RouteUtils", () => {
...
+            ${" "}
+        `(
+            "given queryString is $inputString, returns empty object",
+            ({ inputString }) => {
+                // Arrange & Act
+                const result = RouteUtils.queryStringToObject<any>(inputString);
+
+                // Assert
+                expect(result).not.toBeNull();
+                expect(typeof result).toBe("object");
```
Comment (brandongregoryscott) [inline src/utilities/route-utils.test.ts:232]: Should we assert that it equals `{}`?

File: src/index.ts (hunk 1 above)
Comment (wintondeshong) [reply src/index.ts:1]: Damn 🤦

File: src/utilities/route-utils.test.ts (hunk 2 above)
Comment (wintondeshong) [reply src/utilities/route-utils.test.ts:232]: Yeah, that would be easy way and get more bang for the buck

**Pull request merged**